.github/workflows/release.yml

```diff
@@ -1,36 +1,20 @@
+name: Release sonar-dummy-js
+
 on:
   release:
-    types:
-      - published
+    types: [published]
+  workflow_dispatch:
+    inputs:
+      release_version:
+        description: 'Version'
+        required: false
 
 jobs:
-  publish:
-    runs-on: ubuntu-latest
-    env:
-      RELEASE_TAG: ${{ github.event.release.tag_name }}
-      NPM_REPOSITORY: "sonarsource-npm-public"
-      SCOPE: "%40sonarsource/"
-      PACKAGE: "sonar-dummy-js"
+  releaseasing:
```

Comment:
`releasing` -> `release` same as in sonar-dummy and other projects for coherence